source/manual/2nd-line-drills.html.md

```diff
@@ -72,7 +72,29 @@ See [Communicate when Slack is unavailable](/manual/slack-unavailable.html#2nd-l
 
 ### Deploy from AWS CodeCommit when Github is unavailable
 
-Follow the [Deploy when GitHub is unavailable](/manual/github-unavailable.html#create-and-deploy-an-image-to-ecr) instructions.
+#### Warnings
+
+- Please run the following drill in `integration`. Some steps in the guide below refer to production poweruser, be mindful and choose the correct environment for the drill.
+- Please choose a low impact deployable application that has a Dockerfile. The drill does not cater for repos that are dependencies such as gems and npm packages.
```

Comment:
I don't think we need to refer to gems/NPM packages here? This drill is very much scoped to deploying GOV.UK applications when GitHub is down.

Rubygems/NPM have their own separate deployment pipelines and I can't think of a situation where we'd need to deploy either of those during a GitHub incident.

Reply:
Just wanted to pop this on as an explicit callout to **not** choose gems/npm cause we went for govuk-admin-template, thinking it's the lowest impact 🙈